src/audio/volume/volume.c

```diff
@@ -407,13 +414,14 @@ static struct comp_dev *volume_new(const struct comp_driver *drv,
 
 	reset_state(cd);
 	comp_info(dev, "vol->initial_ramp = %d, vol->ramp = %d, vol->min_value = %d, vol->max_value = %d",
-		  vol->initial_ramp, vol->ramp,
-		  vol->min_value, vol->max_value);
```

Comment:
this should be left untouched as it should be aligned with the open bracket

Reply:
@andrula-song this still hasn't been fixed

Reply:
actually, in my native code i have made it aligned with the open bracket. but the code updated seems not